partcad/src/partcad/shape.py

```
@@ -442,34 +455,44 @@
+                from OCP.StlAPI import StlAPI_Writer
+
                 if not project is None:
                     project.ctx.ensure_dirs_for_file(filepath)
-                cq.exporters.export(
-                    cq_obj,
-                    filepath,
-                    tolerance=tolerance,
-                    angularTolerance=angularTolerance,
-                )
+
+                BRepMesh_IncrementalMesh(obj, tolerance, True, angularTolerance, True)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)